gpMgmt/bin/gplogfilter

```diff
@@ -341,6 +341,15 @@ if begin:
 if end:
     end = end.replace(microsecond=0)
 
+if (begin and end) and begin >= end:
+    execname = os.path.basename(sys.argv[0])
+    print >> sys.stderr, (
+        '%s: Invalid arguments "begin" date is greater or equal'
+        ' than "end" date (%s >= %s)'
+        % (execname, begin, end)
+    )
+    sys.exit(2)
```

Comment:
Maybe I missunderstand you, but there is an similar exit after [catching the IOError exception](https://github.com/arenadata/gpdb/blob/4499c88aef437c7615adf8761c33790d73491eaf/gpMgmt/bin/gplogfilter#L476-L479). Do you mean that exit messages should have the same prefix, or exit should be performed only by try/catch block? 
- I can write append to the exit message IOError
- Or move this check under try/catch block - it seems more readable, but there would be more useless operations performed in case of exceptio

Reply:
I think, It is better to do the same as at all file:
```c
if (begin and end) and begin >= end:
    raise IOError(Invalid arguments "begin" date is greater or equal'
        ' than "end" date (%s >= %s)'
        % (execname, begin, end))
```